### ISSUE TITLE
Backspace not working properly.

### ISSUE THREAD

**Issue by dmitryfedorov292** [jcubic/jsvi#2]
When click backspace it should delete blank line if there are no character anymore

**Comment by jcubic** [comment]
You're right, it should delete the line and move to previous one, will check the code when I have time.

**Comment by dmitryfedorov292** [comment]
Then need to make some change on jsvi lib,right?

**Comment by jcubic** [comment]
yes, that's why I've said that need to check the code and fix it. Or maybe you want to take a look and fix it yourself?

**Comment by dmitryfedorov292** [comment]
It's too difficult for me. could you please help me. I will be appreciated with that
